app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build micropost_params
+    if @micropost.save
+      flash[:success] = t ".success_create"
+      redirect_to root_path
+    else
+      @feed_items = []
+      render root_path
+    end
+  end
+
+  def destroy
+    @micropost.destroy ? flash[:success] = t ".deleted" : flash[:danger] = t ".fail"
+    redirect_to root_url
```

Comment:
So for both of the case(delete/not delete), you want to redirect to the root path (?)

Reply:
Yes, i want to redirect to home  and show flash to notice result